book/_toc.yml

```diff
@@ -24,6 +24,10 @@ parts:
         - file: tutorials/cryocloud_demo/CryoCloud_demo.ipynb
         - file: tutorials/nb-to-package/index
           sections:
+            - file: tutorials/nb-to-package/OOP_intro
+              sections:
+              - file: tutorials/nb-to-package/climate_model.py
+              - file: tutorials/nb-to-package/co2_emissions.py
```

Comment:
```suggestion
```
I don't think these scripts will render. You can add codeblocks to OOP_intro if you want them to show up on the website.

Reply:
I included it because the links in the md don't work otherwise. I'm not too worried about them rendering, I just want a pointer to them.

Reply:
Ah, I see, the CI is just complaining about the scripts not having a title: `OOP_intro.md:1: WARNING: toctree contains reference to document 'tutorials/nb-to-package/climate_model' that doesn't have a title: no link will be generated`